processes/releases.md

```diff
@@ -0,0 +1,42 @@
+# New MeiliSearch releases - what to do
+
+Those are the steps we need to take every time we release a new version of MeiliSearch.
+
+## Responsibilities
+
+Right now, unless mentioned, the tasks for a new release on the developer relations side are divided in three categories or actions with each person accountable for them:
+- Documentation & guides: [@react-learner](https://github.com/react-learner)
+- Blog posts: [@CaroFG](https://github.com/CaroFG)
+- Social media: [@fharper](https://github.com/fharper)
+
+## After the last release
+
+1. Create a PR on a branch called prepare-meilisearch-[YYYY]-[WW] (i.e.: prepare-meilisearch-2020-03 for the third week of the year 2020) in the [documentation repository](https://github.com/meilisearch/documentation).
+1. Ensure that all new PRs related to the next release are done on the mentioned above.
+1. The documentation must contain at minimum the references of the new feature(s), if any.
+1. The decision should be made if we need to create a [guide](https://docs.meilisearch.com/guides/) for the new feature(s), if any.
+
+## Pre-release week
+
+1. If the release has substantial new features, we should draft a [blog](https://blog.meilisearch.com/) post about the new release.
```

Comment:
About the blog post, based a bit on what RUST does when ever they publish a new release: https://blog.rust-lang.org/2020/12/31/Rust-1.49.0.html, I try to be as explanatory as possible. 
For example, about the tokenizer, I would explain a little how it works and what we had before. In order for users to observe that we are evolving. As it took us so long to make that change, it should be at least a bit praised. 

So I would say: 
- new features (may they impact the user or not)
- new addition to the API (clearly showing with example how to use it)
- Breaking changes (for example the default value of the payload limit going from `10mb` to `100mb` is a breaking change that should be communicated).

Those are all what I use to do, it is of course up to you guys to decide if this is a direction you like. 
For the last bit of information, the release article is always popular when they are published. So I think people like to read what is new. So don't be shy :D

Reply:
Added those details, thanks @bidoubiwa